scene/3d/mesh_instance_3d.cpp

```diff
@@ -327,9 +327,6 @@ void MeshInstance3D::create_multiple_convex_collisions(const Ref<MeshConvexDecom
 
 void MeshInstance3D::_notification(int p_what) {
 	switch (p_what) {
-		case NOTIFICATION_ENTER_TREE: {
-			_resolve_skeleton_path();
-		} break;
```

Comment:
I'm not sure this is right, this means setting the path before entering the tree will not have it update here, which will happen for example when instancing a loaded scene

Reply:
What I observed is that the only way to modify the skeleton path is to use the `set_skeleton_path` function, which ensures the skeleton path will always be resolved if it changes. But I'm not sure and need someone to confirm this.

Reply:
But you can set it as a property in a saved scene, and that will be assigned *before* entering the tree

Reply:
Thanks for the detailed explanation. I see what you mean and will try to find a different approach.